src/integTest/java/com/google/cloud/tools/gradle/appengine/AppEngineStandardPluginIntegrationTest.java

```diff
@@ -112,13 +118,6 @@ public void testDevAppServer_async() throws InterruptedException, IOException {
       Thread.sleep(8000);
```

Comment:
Yeah this test is pretty old, not sure why we picked this, probably because at the time it was a lot of work (when is it not?). Anyway, I can try to fix this.